addons/website/views/snippets/s_accordion.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_accordion" name="Accordion">
+<t t-set="uniq" t-value="datetime.datetime.now().microsecond"/>
+    <div class="s_accordion" data-name="Accordion">
+        <div id="myCollapse" t-attf-class="{{extra_classes or 's_accordion_default'}} accordion" role="tablist">
+            <div class="accordion-item position-relative z-1" data-name="Item" role="presentation">
+                <a href="#" class="accordion-header accordion-button justify-content-between gap-2 bg-transparent h6-fs fw-bold text-decoration-none text-reset shadow-none transition-none" t-attf-data-bs-target="#myCollapseTab{{uniq}}_1" role="tab" t-attf-aria-controls="myCollapseTab{{uniq}}_1" data-bs-toggle="collapse" aria-expanded="true"><span class="flex-grow-1">Terms of service</span></a>
+                <div t-attf-id="myCollapseTab{{uniq}}_1" class="accordion-collapse collapse show" data-bs-parent="#myCollapse" role="tabpanel">
+                    <div class="accordion-body">
+                        <p>These terms of service ("Terms", "Agreement") are an agreement between the website ("Website operator", "us", "we" or "our") and you ("User", "you" or "your"). This Agreement sets forth the general terms and conditions of your use of this website and any of its products or services (collectively, "Website" or "Services").</p>
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-item position-relative z-1" data-name="Item" role="presentation">
```

Comment:
Why is `z-1` needed? 🤔

Reply:
Since the "card" template uses a pseudo-element as default background and has a negative z-index (which allows you to customize the card background color on top), this `z-1` ensures that this default background is always visible wherever you drag and drop the snippet.
Example when dropping the accordion in the footer :

| Without z-1 | With z-1 |
|--------|--------|
| <img width="1399" alt="Capture d’écran 2024-07-25 à 09 07 51" src="https://github.com/user-attachments/assets/39f1577d-d760-486f-aeac-0ff6a35ec0ff"> | <img width="1458" alt="Capture d’écran 2024-07-25 à 09 07 30" src="https://github.com/user-attachments/assets/33911c92-84b1-4754-9bd9-8de4ca6eb7ee"> |